system-tests/README.md

```diff
@@ -0,0 +1,10 @@
+# System tests
+
+## Tracing integration tests
+
+The [TracingIntegrationTests](./tests/src/test/java/org/eclipse/dataspaceconnector/system/tests/local/FileTransferIntegrationTest.java) makes sure that tracing works correctly when triggering a file transfer.
+To be able to run the `TracingIntegrationTests` locally, you need to place the [opentelemetry java agent](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v1.12.0/opentelemetry-javaagent.jar) in the project root folder.
+Then you can run the tests:
+```bash
+./gradlew -p system-tests/tests test -DincludeTags="OpenTelemetryIntegrationTest"
+```
```

Comment:
It would be good to add details on how and why the `OtlpGrpcServer` is used for these tests.